quickwit/quickwit-serve/src/ingest_api/rest_handler.rs

```diff
@@ -91,12 +86,14 @@ struct BulkActionMeta {
     id: String,
 }
 
+fn bind<T: Clone + Send>(value: T) -> impl Filter<Extract = (T,), Error = Infallible> + Clone {
```

Comment:
we can probably make it `pub(crate)` and put it elsewhere (can handle this in another PR)